### DISCUSSION TITLE
Switching between start and start-dev deletes data?

### DISCUSSION THREAD

**Discussion by nihiluis** [keycloak/keycloak#20365] [Q&A]
I was using the docker image with command start-dev and switched later to start. Then I switched back to start-dev and found that my realm was deleted. Why?

**Comment by sschu** [comment]
Did you use an external persistent database? Because start-dev normally uses an in-memory database that is gone once you stop Keycloak.

**Reply by nihiluis** [reply]
I had DB_VENDOR env set as POSTGRES (from an earlier version) and now set it to postgres. I think this might be the culprit. I will see if it gets deleted again.
